app/controllers/concerns/course/forum/topic_controller_subscription_concern.rb

```diff
@@ -0,0 +1,39 @@
+module Course::Forum::TopicControllerSubscriptionConcern
+  extend ActiveSupport::Concern
+
+  def subscribe
+    authorize!(:read, @topic)
```

Comment:
hmm, another question: if I do this, in 'subscribe' ability, I just need to define permissions that are not included in 'read' right?